packages/compass-components/src/components/interactive-popover.tsx

```diff
@@ -167,7 +184,7 @@ function InteractivePopover({
           focusTrapOptions={{
             clickOutsideDeactivates: true,
             // Tests fail without a fallback. (https://github.com/focus-trap/focus-trap-react/issues/91)
-            fallbackFocus: `#${closeButtonId}`,
+            fallbackFocus: customFocusTrapFallback || `#${closeButtonId}`,
```

Comment:
NIT: do we need a custom fallback? it seems it was only needed for the tests of the popover, probably i'm wrong though. if you don't specify anything it should focus on the first focusable element on its own

Reply:
TBH, the main reason is testing, the whole plugin fails if not. Another option I had earlier was an empty invisible component when there is no closing button, however, it's even worse.

However, with this behaviour, right now, if someone changes the interactive popover that we are using and doesn't add a focusable component, it will focus on the close button, instead of breaking the UI, which is more resilient. For example, if for some reason we disable the input until we load something (for example to check existing names to avoid overwriting) or any other behaviour we might think of.